bin/format_secrets.rb

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
```

Comment:
I think the name of the file should be more explicit that it's formatting secrets only the AWS on the CD flow. It's not a generic formatting script